plugin.audio.tilos/addon.py

```diff
@@ -0,0 +1,507 @@
+"""
+    Plugin to browse the archive and listen to 
+    live stream of Radio Tilos
+"""
+
+import sys, os, os.path, xbmcaddon
+import xbmc, xbmcgui, xbmcplugin
+from urllib2 import Request, urlopen, URLError
+import urlparse
+import urllib
+from HTMLParser import HTMLParser
+from shutil import rmtree, copy
+import traceback
+from pprint import pprint
+import json
+import time
+import calendar
+import datetime
+import re
+
+
+############################################
+# Define required statics
+############################################
+
+__plugin__ = "Tilos"
+__version__ = '0.0.5'
+__author__ = 'Gabor Boros'
+__date__ = '2014-07-28'
+__addon__ = xbmcaddon.Addon()
```

Comment:
In your next update please change to not use the \__labelname\__ style labels. This should only be used for python internals.